lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -29,16 +29,23 @@ defmodule Trento.Clusters do
 
   @spec request_checks_execution(String.t()) :: :ok | {:error, any}
   def request_checks_execution(cluster_id) do
-    ClusterReadModel
-    |> Repo.get(cluster_id)
+    query =
```

Comment:
We don't need this assignment, right?
Cannot we pipe this with `Repo.one` already?
What is the `first` usage there by the way?
Anyway, I guess it comes from other functions in the file that do the same